src/content/docs/pt-br/basics/astro-pages.mdx

```diff
@@ -62,8 +66,7 @@ import LayoutDoMeuSite from '../layouts/LayoutDoMeuSite.astro';
   <p>O conteúdo da minha página, envolto em um layout!</p>
 </LayoutDoMeuSite>
 ```
-
-📚 Leia mais sobre [componentes de layout](/pt-br/basics/layouts/) no Astro.
+<ReadMore>Leia mais sobre [componentes de layout](/pt-br/basics/layouts/) no Astro.</ReadMore>
```

Comment:
Same here

```suggestion

<ReadMore>Leia mais sobre [componentes de layout](/pt-br/basics/layouts/) no Astro.</ReadMore>
```

Reply:
Oops, I'm getting "Applying suggestions on deleted lines is currently not supported.", is that something that should be fixed on my end?

Reply:
Not on your end, just GH being weird